chips/sam4l/src/pm.rs

```diff
@@ -259,243 +148,170 @@ pub enum SystemClockSource {
     /// Use the internal digital frequency locked loop (DFLL) sourced from
```

Comment:
These enums now seem to imply that you can only use each clock at a fixed frequency. Long term that's probably not exactly right, but if nothing else we should update to comment on `SystemClockSource` to make it clear what's going on.